Wrong code

surajmb opened the pull request:
In the Disabled rules section, we have given CLI commands whereas this page is for PowerShell commands. I've updated it with the right commands.

azurecla commented:
Hi __@SSalahAhmed__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

surajmb committed:
Wrong code

In the Disabled rules section, we have give CLI commands whereas this page is for PowerShell commands. I've updated it with the right commands.
